smart_campus/app/migrations/0001_initial.py

```diff
@@ -1,9 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
Again, we don't version control migration files when development.
Please ***DO NOT*** use `git add .`.
Also, you should ***ALWAYS*** use `git status --staged` before you commit a change.

Reply:
okay, my fault.